gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CheckImplicitDependenciesTask.java

```diff
@@ -142,7 +142,7 @@ private boolean shouldIgnore(ResolvedArtifact artifact) {
         return ignore.get().contains(BaselineExactDependencies.asString(artifact));
     }
 
-    @Input
+    @InputFiles
```

Comment:
should this be `@Classpath`?

Reply:
good catch, yep that will fix any ordering issues!